modules/setting-up-vmc-for-vsphere.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vmc/installing-restricted-network-vmc-user-infra.adoc
+// * installing/installing_vmc/installing-vmc-customizations.adoc
+// * installing/installing_vmc/installing-vmc-network-customizations.adoc
+// * installing/installing_vmc/installing-vmc-network-customizations-user-infra.adoc
+// * installing/installing_vmc/installing-vmc-user-infra.adoc
+// * installing/installing_vmc/installing-vmc.adoc
+
+[id="setting-up-vmc-for-vsphere_{context}"]
+= Setting up VMC for vSphere
+
+You can install {product-title} on VMware Cloud (VMC) on AWS hosted vSphere clusters to enable applications to be deployed and managed both on-premises and off-premises, across the hybrid cloud.
+
+image::vmc-illustration.png["VMC on AWS Architecture"]
```

Comment:
This diagram shows RWX which isn't supported without vSAN File Services, which as far as I can find unavailable in VMC.

Reply:
@acsulli sounds like we should pull RWX from this diagram?

Reply:
Makes sense to me.